### PR TITLE
add upgrade instructions

### PR DESCRIPTION
**Pull request by ali-ince**
This PR adds basic upgrade instructions.

### CODE REVIEW

File: modules/ROOT/pages/upgrade/index.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+= Upgrading from an earlier version
+:description: This section describes the upgrade process from an earlier version of NOM.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/upgrade/index.adoc:2]: Switch places, description first, then header.

File: modules/ROOT/pages/upgrade/index.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+[[server]]
+== Upgrading server
+
+[NOTE]
+====
+Before starting the upgrade process, it is recommended to backup your existing NOM installation folder.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/upgrade/index.adoc:9]:
```suggestion
Before starting the upgrade process, it is recommended to back up your existing NOM installation folder.
```

File: modules/ROOT/pages/upgrade/index.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+====
+
+[[server-linux]]
+=== On Linux
+
+. Backup any files you've modified inside the current NOM server directory.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/upgrade/index.adoc:15]:
```suggestion
. Back up any files you've modified inside the current NOM server directory.
```

File: modules/ROOT/pages/upgrade/index.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+. Stop NOM server by running `systemctl stop neo4j-ops-manager-server`.
+. Extract the downloaded package into the existing installation directory.
+. Copy the backed up files back into their corresponding locations.
+. Start NOM server by running `systemctl start neo4j-ops-manager-server`.
+. Verify that the server is operating as expected by checking logs.
+
+[[server-windows]]
+=== On Windows
+
+. Backup any files you've modified inside the current NOM server directory.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/upgrade/index.adoc:27]:
```suggestion
. Back up any files you've modified inside the current NOM server directory.
```

**Review by AlexicaWright — COMMENTED** [review]
Just some editorial comments. Thank you @ali-ince !

**Review by AlexicaWright — APPROVED** [review]
👍 LGTM!

**Review by eve-bright — APPROVED** [review]

**Pull request merged**